### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by AbdelrhmanWalaa**
Add "RGB LED Control V2.0" Task

### CODE REVIEW

File: 10 - ARM/S_ARM_03 RGB LED Control V2.0/Project - RGB LED Control V2.0/led_program.c [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+{
+	/* Define local variable to set the error state = OK. */
+	LED_enErrorState_t Loc_enErrorState = LED_EN_OK;
+		
+	/* Check 1: LEDId and Operation are in the valid range. */
+	if ( ( u8_a_ledId <= LED_U8_7 ) 			      &&
+		   ( u8_a_operation <= LED_U8_TOGGLE ) )
+	{
+		/* Check 1.1: Required Operation. */
+		switch ( u8_a_operation )
```
Comment (Hala-Abdelkader) [inline 10 - ARM/S_ARM_03 RGB LED Control V2.0/Project - RGB LED Control V2.0/led_program.c:87]: Switch case should have only one break

**Review by Hala-Abdelkader — COMMENTED** [review]
-There is no flowchart for application in your pdf
-Your work is really perfect, thank you for your effort.

**Comment by AbdelrhmanWalaa** [comment]
> 

Thanks a lot for your support.
Application flowcharts will be added ASAP.

**Comment by Hala-Abdelkader** [comment]
Don't waste your time doing application flowcharts for this task now as it is just a comment to take care in the next deliveries.

Also the comment of switch case, apply it in the next deliveries.

**Comment by AbdelrhmanWalaa** [comment]
> Don't waste your time doing application flowcharts for this task now as it is just a comment to take care in the next deliveries.
> 
> Also the comment of switch case, apply it in the next deliveries.

I appreciate the feedback and understand the areas where I need to improve. I assure you that I will actively work on avoiding these mistakes and will be more cautious and attentive going forward.

**Pull request merged**